.github/workflows/release.yml

```
@@ -38,3 +38,77 @@ jobs:
         if: ${{startsWith(github.ref, 'refs/tags/') }}
         with:
           files: pie.phar
+
+  docker-binary-only-image:
+    needs: build-phar
+    name: Docker binary-only image
+    runs-on: ubuntu-latest
+    if: ${{ startsWith(github.ref, 'refs/tags/') }}
+
+    permissions:
+      # attestations:write is required to publish provenance attestations for the built Docker image
+      attestations: write
+      # id-token:write is required for logging in to the Docker registry
+      id-token: write
```

```suggestion
      # id-token:write is required for build provenance attestation.
      id-token: write
      # attestations:write is required for build provenance attestation.
      attestations: write
```

Taken from: https://github.com/php/pie/blob/main/.github/workflows/build-phar.yml

---

I've applied changes to descriptions, but kept permissions ordered.